cmd/controller/main.go

```diff
@@ -104,11 +103,9 @@ func main() {
 	clusterBuildTemplateInformer := buildInformerFactory.Build().V1alpha1().ClusterBuildTemplates()
 	imageInformer := cachingInformerFactory.Caching().V1alpha1().Images()
 
-	bldr := onclusterbuilder.NewBuilder(kubeClient, kubeInformerFactory, logger)
-
 	// Build all of our controllers, with the clients constructed above.
-	controllers := []controller.Interface{
-		build.NewController(logger, kubeClient, buildClient, buildInformer, buildTemplateInformer, clusterBuildTemplateInformer, bldr),
+	controllers := []*controller.Impl{
+		build.NewController(logger, kubeClient, kubeInformerFactory, buildClient, buildInformer, buildTemplateInformer, clusterBuildTemplateInformer),
```

Comment:
The pattern we're using constructs the Pod informer here and passes it in, so that it can be included in the synchronization below.